extensions/notebook/package.json

```diff
@@ -507,27 +525,6 @@
           "connectionProviderIds": []
         }
       ]
-    },
-    "viewsContainers": {
-      "activitybar": [
-        {
-          "id": "books-explorer",
-          "title": "Notebooks",
-          "icon": "resources/dark/JupyterBook_2.svg"
```

Comment:
Can this be deleted?

Reply:
Previously I was adding the viewlet from extension, that's why this is needed. But now since I created a notebooks explorer in core, extension just contributes contents as views to the viewlet. So this needs to be cleaned up.

Reply:
I'm saying that you should delete the SVG if it isn't used anywhere else

Reply:
gotcha, will cleanup.